### PR TITLE
Fix: Set JTI attribute in JWT to same value as ID attribute of VP

### LINKED ISSUE #80: VP token does not comply with VC Data Model
The VC Data Model (see [JWT Encoding](https://www.w3.org/TR/vc-data-model/#jwt-encoding) stated that the **jti** claim of the JWT must have the same value as the **id** attribute of the respective VP. However, currently the **jti** claim is a random UUID.

### PR DESCRIPTION
**Pull request by bjoern-arnold**
## Description
Fixes #80 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by bjoern-arnold** [comment]
@borisrizov-zf could you please also consider this fix for the next release? It turned out that due to this issue the VP tokens cannot be verified by other libraries (e.g. [Sphereon PEX](https://github.com/Sphereon-Opensource/PEX/).

**Comment by borisrizov-zf** [comment]
@bjoern-arnold Yes, no problem. We can merge this after #39 (you'll need to rebase this branch after #39 is merged).

**Comment by bjoern-arnold** [comment]
@borisrizov-zf after you merged #39 I rebased my branch as requested.

**Review by borisrizov-zf — APPROVED** [review]

**Pull request merged**
